gh-actions/common/build-debian/action.yml

```diff
@@ -140,6 +147,9 @@ runs:
           fi
           echo "::endgroup::"
 
+          # FIXME: This part should moved to ${{ inputs.extra-source-build-script }}
+          # Since it's very project-specific, as most of projects don't care about
+          # Having git at all when building the package source.
           if command -v git &> /dev/null; then
             git config --system --add safe.directory "${{ github.workspace }}"
           fi
```

Comment:
This actually seems like a quite generic thing to me (ensuring that git can be executed in the workspace directory even when it has a different owner, when git is installed)

Reply:
Well this is maybe true... But the rest of things (like the github token looks quite per-repo specific IMHO).

Reply:
> But the rest of things (like the github token looks quite per-repo specific IMHO)

does it? it configures git to use the `GITHUB_TOKEN` in its URL. that doesn't look repo-specific to me.

Reply:
Yeah, that's fine... I've dropped the FIXME already and we do install git when that's defined :)